scss/footer.scss

```diff
@@ -0,0 +1,30 @@
+@import 'colors';
+@import 'sizes';
+
+.footer {
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  height: 200px;
+  width: 100%;
+}
+
+.footer-copyright {
+  padding-top: 30px;
```

Comment:
this should be a margin bottom in the image since you want siblings to separate from each other